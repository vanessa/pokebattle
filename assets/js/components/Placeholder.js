import React from 'react';

export default function Placeholder() {
  return (
    <div className="placeholder" />
  );
}
