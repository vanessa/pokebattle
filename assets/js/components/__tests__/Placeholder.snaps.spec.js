import React from 'react';
import renderer from 'react-test-renderer';
import Placeholder from '../Placeholder';

describe('Placeholder', () => {
  let Component;
  let tree;

  test('renders', () => {
    Component = renderer.create((
      <Placeholder />
    ));

    tree = Component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
