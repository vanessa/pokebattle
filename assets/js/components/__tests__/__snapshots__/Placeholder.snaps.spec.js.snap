// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder renders 1`] = `
<div
  className="placeholder"
/>
`;
